scriptbuilder/pom.xml

```diff
@@ -83,6 +83,6 @@
         </configuration>
       </plugin>
     </plugins>
-  </build>
+  </build>-->
```

Comment:
I'm not an OSGi expert, but could you please explain why this should be commented?